definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -597,105 +492,105 @@
   {
     "LiveFrom": "05/08/2020",
     "CaseTypeID": "DIVORCE",
-    "CaseFieldID": "PBAReferenceLabel",
+    "CaseFieldID": "LabelPBAReference",
```

Comment:
New authorisations wouldn't be needed if we use the existing label fields.

Reply:
Actually, just noticed those existing label fields are only for the solicitor.
Would need the new fields then unless we extend the solicitor label fields